Beginner_Projects/captionTags/app.py

```
@@ -0,0 +1,43 @@
+from flask import Flask, render_template, request
+import os
+from dotenv import load_dotenv
+import google.generativeai as genai
+import re
+
+load_dotenv()
+gemini_api_key = os.getenv("GOOGLE_API_KEY")
+genai.configure(api_key=gemini_api_key)
+app = Flask(__name__)
+
+@app.route("/", methods=["GET", "POST"])
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)